paddle/gserver/evaluators/ChunkEvaluator.cpp

```diff
@@ -243,6 +242,53 @@ class ChunkEvaluator : public Evaluator {
     if (tag == tagSingle_) return true;
     return false;
   }
+
+public:
+  // three metrics: precision, recall and F1-score
+  void getNames(std::vector<std::string>* names) {
+    this->storeLocalValues();
+    names->reserve(this->values_.size());
```

Comment:
change it to `names->reserve(names.size() + values_.size())`, because `names` may be nonempty.